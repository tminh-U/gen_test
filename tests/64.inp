10
17 7 18 3 19 15 20 13 12 10 
15
9 10
7 9
3 3
9 10
5 7
5 10
4 4
10 10
8 9
2 10
5 9
2 3
3 10
1 7
1 3
