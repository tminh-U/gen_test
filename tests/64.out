3
4
2
3
6
9
2
2
3
35
8
4
22
34
6
